components/brave_education/resources/brave_education.html

```diff
@@ -0,0 +1,36 @@
+<!doctype html>
+<html dir="$i18n{textdirection}" lang="$i18n{language}">
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width">
+  <title>$i18n{headerText}</title>
+  <style>
+    html {
+      --page-background: $i18n{backgroundColor};
+    }
+  </style>
+  <style>
+    html {
+      background: var(--page-background);
+      height: 100%;
+      overflow: hidden;
+    }
+
+    body {
+      height: 100%;
+      margin: 0;
+    }
+
+    #content {
+      border: none;
+      height: 100%;
+      width: 100%;
+    }
+  </style>
+  <script type="module" src="/strings.m.js"></script>
+  <script type="module" src="/brave_education.bundle.js"></script>
+</head>
+<body>
+  <iframe id="content" title="Brave product education"></iframe>
```

Comment:
this needs a `sandbox` attribute. you can allow attributes as needed but start from just `allow-scripts`.